data/dsql/sql.go

```diff
@@ -1,9 +1,14 @@
-package data
+// Package dsql provides helpers for scanning database/sql responses into
```

Comment:
Package naming is hard. dsql is probably the best we can do, but what does d denote, data? 

Reading package description would make more sense with sqlhelper or sqlutil. To denote sql helpers for data.frame maybe sql frame, but I don't know 😄

Reply:
I agree with @marefr